Cargo.toml

```diff
@@ -71,7 +71,7 @@ version = "1.0"
 features = ["arbitrary_precision", "unbounded_depth"]
 
 [dependencies.secp256k1]
-version = "0.24.2"
+version = "0.24.3"
```

Comment:
Should we really specify a full rev, not just `0.24.`?

Reply:
The only risk in doing so is a very minor nuisance: If someone runs `cargo update` in an unrelated PR, it will make their PR messy because the dependency moves.

Barring that, I'm happy to leave the patch version unpinned in our .toml files.

Reply:
> The only risk in doing so is a very minor nuisance: If someone runs `cargo update` in an unrelated PR, it will make their PR messy because the dependency moves.
> 
> Barring that, I'm happy to leave the patch version unpinned in our .toml files.

Probably safe to leave full rev for now, we dependabot should tell us if there's a problem.